stan/math/prim/scal/prob/chi_square_ccdf_log.hpp

```diff
@@ -93,7 +93,7 @@ namespace stan {
         const T_partials_return beta_dbl = 0.5;
 
         // Compute
```

Comment:
I didn't want a 600 file diff, so I'm taking it in smaller chunks. Hopefully review is something like 10 mins or less each and we'll eventually get through it all.